### ISSUE TITLE
 Got the Top-1 accu of only 70.7% when training efficientnet-b3 with Imagenet

### ISSUE THREAD

**Issue by YvetteLaw** [lukemelas/EfficientNet-PyTorch#301]
I train efficientnet-b3 with ImageNet dataset by the script in ./examples/imagenet/main.py with the following command 
`python main.py /path/to/imagenet -b 128 --image_size 300 -a 'efficient-b3' `. The GPU of my machine is Tesla V100 and I trained with 8P. After the default 90 epochs, I got the following result: Acc@1 70.702 Acc@5 90.050. The top-1 accuracy of 70.7% is not as good as 80.8% claimed.   Is there anything wrong with my command? Or what factors result in this result？

**Comment by RuiFeiHe** [comment]
I eval the provided efficientnet-b4, and got only 77.516 top1 on imagenet val set.

**Comment by JGyoung33** [comment]
> I eval the provided efficientnet-b4, and got only 77.516 top1 on imagenet val set.

I got only 74.99% top1 with efficientnet-b4.

**Comment by hamzakhalil798** [comment]
@YvetteLaw  can you please share the dataset structure?
